packages/api-clients/template-bff.hbs

```diff
@@ -86,6 +86,7 @@ export const {{@key}}Api = new Zodios({{#if options.baseUrl}}"{{options.baseUrl}
 export function create{{pascalcase @key}}ApiClient(baseUrl: string, options?: ZodiosOptions) {
     return new Zodios(baseUrl, {{@key}}Endpoints, {
 			...options,
+			validate: false,
```

Comment:
This change disables the validation before sending a request using a Zodios client. The validation will be performed server-side only.